AEPCore/Sources/rules/RuleConsequence.swift

```diff
@@ -22,6 +22,18 @@ public struct RuleConsequence {
         return details["eventdata"] as? [String: Any?]
     }
 
+    public var eventSource: String? {
```

Comment:
nit: These properties only apply to dispatch consequence. Can we move these into a separate extension with comments?